nodejs/week3/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "week3",
+  "version": "1.0.0",
+  "description": "",
```

Comment:
If you use ES6 imports you would have to include the line `"type": "module",` in this file, or use the `.mjs` file extension instead of `.js`.